addons/containerd/1.4.4/Manifest

```diff
@@ -1,6 +1,9 @@
-dockerout rhel-7 addons/containerd/template/Dockerfile.centos7 1.4.4
-dockerout rhel-7-force addons/containerd/template/Dockerfile.centos7-force 1.4.4
-dockerout rhel-8 addons/containerd/template/Dockerfile.centos8 1.4.4
+dockerout centos-7 addons/containerd/template/Dockerfile.centos7 1.4.4
+dockerout centos-7-force addons/containerd/template/Dockerfile.centos7-force 1.4.4
+dockerout centos-8 addons/containerd/template/Dockerfile.centos8 1.4.4
+dockerout rhel-7 addons/containerd/template/Dockerfile.rhel7 1.4.4
+dockerout rhel-7-force addons/containerd/template/Dockerfile.rhel7-force 1.4.4
```

Comment:
i dont think there is any reason to build this bundle and itll just make the packages even bigger. same goes for all instances of rhel-7-force in this pr

Reply:
oh we do it for centos because we use that for amazon ol?